creator-node/src/services/sync/secondarySyncFromPrimary.js

```diff
@@ -49,14 +49,21 @@ const handleSyncFromPrimary = async ({
   /**
    * Perform all sync operations, catch and log error if thrown, and always release redis locks after.
    */
-  const forceResync = await shouldForceResync(
+  const shouldForceResyncBool = await shouldForceResync(
```

Comment:
can we actually keep the original variable name? I dont think we need to add to bool as it's implied here

Reply:
ya fair